docs/get-started/sumo-logic-ui.md

```diff
@@ -1,115 +1,135 @@
+    * **Cloud SIEM Integrations**. Manage [Ingest Mappings](/docs/cse/ingestion/sumo-logic-ingest-mapping), [Log Mappings](/docs/cse/schema/create-structured-log-mapping), [Context Actions](/docs/cse/administration/create-cse-context-actions), [Actions](/docs/cse/administration/create-cse-actions), [Enrichment](/docs/cse/integrations/enrichments-and-indicators), [Automation](/docs/cse/automation).
+    * **Cloud SIEM Entities**. Manage [Groups](/docs/cse/records-signals-entities-insights/create-an-entity-group), [Normalization](/docs/cse/schema/username-and-hostname-normalization), [Custom Types](/docs/cse/records-signals-entities-insights/create-custom-entity-type), and [Criticality](/docs/cse/records-signals-entities-insights/entity-criticality).
+    * **Cloud SIEM Workflow**. Manage [Insight Detection](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold), [Insight Statuses](/docs/cse/administration/manage-custom-insight-statuses), [Insight Resolutions](/docs/cse/administration/manage-custom-insight-resolutions), and [Tag Schemas](/docs/cse/administration/create-a-custom-tag-schema).
 
-1. Hover the cursor over the tab **Details** icon.
+### Manage accounts, users, and security
 
-    ![WTS_Tab-options-details.png](/img/get-started/ui/WTS_Tab-options-details.png)
+Sumo Logic administrators (admins) manage user accounts, user roles, and security. You must have Sumo Logic Admin role privileges to perform these tasks.
 
-1. Select the tab and choose an option from the dropdown menu.
+To administer Sumo Logic accounts, users, and security, do the following:
+
+1. From the top navigation bar, click **Administration**.<br/><img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin.png" width="400"/>
+1. Choose from any of the following:
+    * **Account**. [View information about your organization's Sumo Logic subscription](/docs/manage/manage-subscription), [enable and manage the data volume index,](/docs/manage/ingestion-volume/data-volume-index) [manage billing](/docs/manage/manage-subscription).
+    * **Users and Roles**. [Manage users and roles](/docs/manage/users-roles).
+    * **Account Security Settings**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](/docs/dashboards/share-dashboard-new/), and [set up SAML authentication](/docs/manage/security/saml).<br/><img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin-expanded.png" width="400"/>
```

Comment:
```suggestion
    * **Account Security Settings**. [Set password policy for your org](/docs/manage/security/set-password-policy), [set up security whitelist](/docs/manage/security/create-allowlist-ip-cidr-addresses), [manage access keys](/docs/manage/security/access-keys), manage security policies ([audit index](/docs/manage/security/audit-indexes/audit-index)), [support account access](/docs/manage/security/enable-support-account), [dashboard sharing](/docs/dashboards/share-dashboard-new/), and [set up SAML authentication](/docs/manage/security/saml).<br/><img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin-expanded.png" width="400"/>
    * **Cloud SOAR Settings**. Manage [general settings](/docs/cloud-soar/global-functions-menu/#general-settings), [Notifications](/docs/cloud-soar/global-functions-menu/#notifications), and [Groups](/docs/cloud-soar/global-functions-menu/#groups).
```

Reply:
@kimsauce, I will go ahead and commit this change to add "Cloud SOAR Settings" so we don't forget about it.

Reply:
thank you!